### PR TITLE
kik: updated description

### PR DESCRIPTION
**Pull request by austinhallock**

### CODE REVIEW

File: server.coffee [hunk 1]
```diff
@@ -209,6 +208,15 @@ renderHomePage = do ->
   kikPage = _.defaults
     isProbablyKik: true
     title: 'Free Games'
+    description: 'Play fun mobile games instantly on your phone for free. ' +
+                 'Clay has the best games.' +
+                 (_.map _.range(108), -> '&nbsp;').join('') +
```
Comment (Zolmeister) [inline server.coffee]: magic number, and should be calculated relative to current value

**Comment by austinhallock** [comment]
updated (note that we are suppressing escaping for descriptions) http://jimdowney.net/2013/06/10/the-many-ways-of-escaping-dust-js-filters-2/

**Comment by austinhallock** [comment]
also note that kik might consider &nbsp; as 6 chars. If so we'll just have to come up with a longer normal description

File: server.coffee [hunk 2]
```diff
@@ -206,9 +206,21 @@ renderHomePage = do ->
...
+  kikDescription = 'Play fun mobile games instantly on your phone for free.'
+  kikKeywords = 'snapchattinderfacebooktwittergoogleyoutubevineinstagram' +
+                'redditheyheymeetkiktwitterpinterestappstumblrnetflixhulu' +
+                'itunesfeedxboxpornsexyfriendschatboysgirlscutequizbuzzcat' +
+                'strangersdogspussymocoboobsgaylesbianloveroleplaymixbored' +
+                'flirtygifhottestkimbeyoncetaylorluckyjustinbiebermileykaty' +
+                'selenanickicandyangryflappy'
+  kikWhitespaceLength = KIK_DESCRIPTION_MAX_CHARS - kikDescription.length -
+                        kikKeywords.length
+  kikWhitespace = (_.map _.range(kikWhitespaceLength), -> '&nbsp;').join('')
```
Comment (Zolmeister) [inline server.coffee]: use `_.repeat` (may only be in lodash v3)

File: test/server.coffee [hunk 3]
```diff
@@ -104,6 +104,18 @@ describe 'index.dust', ->
         .flare (flare) ->
           flare.res.body.should.contain '<title>Free Games</title>'
 
+    it 'Should include 500 character description', ->
+      flare
+        .get '/'
+        .flare (flare) ->
+          findBefore = '<meta name="description" content="'
```
Comment (Zolmeister) [inline test/server.coffee]: this is gross, use a regex

**Comment by Zolmeister** [comment]
we can't not escape the description. try using the literal space: ` `

**Comment by Zolmeister** [comment]
done

**Comment by austinhallock** [comment]
https://github.com/claydotio/clay-mobile/commit/a89cf6ad52216ebf67ea1a1fae9887539fcb6154

**Comment by Zolmeister** [comment]
https://github.com/claydotio/clay-mobile/pull/197#issuecomment-71702328

**Comment by austinhallock** [comment]
https://github.com/claydotio/clay-mobile/commit/1bdf59225b61ea3986f195b666c54d5356fea774

**Comment by Zolmeister** [comment]
tests

**Comment by Zolmeister** [comment]
rebase

**Comment by austinhallock** [comment]
done

**Comment by Zolmeister** [comment]
`server:`

**Pull request merged**